e2e/pages/nav-page.ts

```diff
@@ -36,7 +36,8 @@ export class NavPage {
     this.page = page;
     this.mainTitle = page.locator('[data-test="mainTitle"]');
     this.progressBar = page.locator('.progress');
-    this.preferencesButton = page.locator('.preferences-button-nav button');
+    // this.preferencesButton = page.locator('[data-test="preferencesButton"]');
```

Comment:
Please don't leave commented code around (just delete it).

Reply:
Oops, I have just removed the commented line of code.